README.md

```diff
@@ -146,6 +146,17 @@ pip3 install -r sphinx/requirements.txt
 python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 ```
 
+### Improving performance on MI300 when using less than 8 GPUs
+
+On a system with 8\*MI300X GPUs, each pair of GPUs are connected with dedicated XGMI links in a fully-connected topology. So, for collective operations, one can achieve good performance when all 8 GPUs (and all XGMI links) are used. When using less than 8 GPUs, one can only achieve a fraction of the potential bandwidth on the system.
+
+But, if your workload warrants using less than 8 MI300 GPUs on a system, you can set the run-time variable `NCCL_MIN_NCHANNELS` to increase the number of channels.\
+E.g.: `export NCCL_MIN_NCHANNELS=32`
+
+Increasing the number of channels can be beneficial to performance, but it also increases GPU utilization for collective operations.
+
+Additionally, we have pre-defined higher number of channels when using only 2 GPUs or 4 GPUs on a 8\*MI300 system. Here, RCCL will use **32 channels** for the 2 MI300 GPUs scenario and **24 channels** for the 4 MI300 GPUs scenario.
```

Comment:
> Additionally, there are a pre-defined higher number of channels used when using only 2 or 4 of the GPUs on a 8\*MI300 system. RCCL will use **32 channels** with 2 MI300 GPUs and use **24 channels** with 4 MI300 GPUs.

I can't tell if it's that RCCL **sets** or **uses** a pre-defined number of channels when using 2 or 4 GPUs. I put in "use", but if its sets (or something else), just put that in. If it's something else, please let me know so I understand for next time.